<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>468</width>
    <height>597</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1,1">
    <item>
     <widget class="QGroupBox" name="paramsGroup">
      <property name="title">
       <string>Parameters</string>
      </property>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Initialisation Vector:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="inputIV"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Key:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="inputKey"/>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="inputGroup">
      <property name="title">
       <string>Input</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QPlainTextEdit" name="inputPlainText"/>
       </item>
       <item>
        <widget class="QCheckBox" name="cbHex">
         <property name="text">
          <string>Input is in hex</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="encryptButton">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Encrypt</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="outputGroup">
      <property name="title">
       <string>Output</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QPlainTextEdit" name="outputCiphertext">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="copyButton">
         <property name="text">
          <string>Copy</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>468</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>inputIV</sender>
   <signal>textChanged(QString)</signal>
   <receiver>MainWindow</receiver>
   <slot>updateControlState()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>285</x>
     <y>69</y>
    </hint>
    <hint type="destinationlabel">
     <x>233</x>
     <y>298</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>inputKey</sender>
   <signal>textChanged(QString)</signal>
   <receiver>MainWindow</receiver>
   <slot>updateControlState()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>285</x>
     <y>97</y>
    </hint>
    <hint type="destinationlabel">
     <x>233</x>
     <y>298</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>inputPlainText</sender>
   <signal>textChanged()</signal>
   <receiver>MainWindow</receiver>
   <slot>updateControlState()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>233</x>
     <y>228</y>
    </hint>
    <hint type="destinationlabel">
     <x>233</x>
     <y>298</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>encryptButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>encrypt()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>126</x>
     <y>319</y>
    </hint>
    <hint type="destinationlabel">
     <x>233</x>
     <y>298</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>copyButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>copyToClipboard()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>233</x>
     <y>543</y>
    </hint>
    <hint type="destinationlabel">
     <x>233</x>
     <y>298</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbHex</sender>
   <signal>clicked(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>setHexMode(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>233</x>
     <y>293</y>
    </hint>
    <hint type="destinationlabel">
     <x>233</x>
     <y>298</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>updateControlState()</slot>
  <slot>encrypt()</slot>
  <slot>copyToClipboard()</slot>
  <slot>slot1()</slot>
  <slot>setHexMode(bool)</slot>
 </slots>
</ui>
